canonical/vanilla-monorepo
# feat: Base storybook

Adds a base storybook package.

## QA

1. Checkout this PR
2. Install dependencies: `bun install`
3. Open the DS package: `cd packages/ds`
4. Run storybook: `npm run storybook` (running it with bun or yarn intermittently causes errors on some pages)
5. Open [storybook](http://0.0.0.0:6006/)
6. Navigate around storybook, see that all the routes load and the base installation works
7. Open any story, IE [primary button](http://0.0.0.0:6006/?path=/story/example-button--primary). Verify that it has a theme switcher at the top of the page, and that switcher changes the theme class on the story's iframe to the relevant Vanilla CSS theme class (`is-dark`, `is-paper`, `is-light`). No visual changes will occur yet, as we haven't yet included Vanilla's styles.
8. Open the [react logo](http://0.0.0.0:6006/assets/react.svg) to verify that files in `src/assets` are exposed publicly under the `assets` directory.
